### PR TITLE
Fix connected calendar list

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Fixed an issue with the calendar list where an error showed up regarding primary calendar, which was referring to non-calendar apps not being filtered correctly.

Before:
<kbd></kbd>

After: 
<kbd></kbd>

**Environment**: Staging(main branch) / Production

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Have Hubspot installed along with a Google Calendar (with or without proper permissions.

### CODE REVIEW

**Review by emrysal — APPROVED** [review]
This fixes the immediate issue but we need context-based errors in the future.

🚀

**Comment by alannnc** [comment]
> This fixes the immediate issue but we need context-based errors in the future.
> 
> 🚀

I'm working on a Fix for this. Catch the error and return connectedCalendar with an error flag. We can give this to production while I work on my solution as it will require a lot of changes.

**Review by zomars — APPROVED** [review]
Ship it

File: playwright.config.ts [hunk 1]
```diff
@@ -59,6 +59,10 @@ const config: PlaywrightTestConfig = {
       timeout: 120000,
       use: {
         ...devices["Desktop Chrome"],
+        viewport: {
+          width: 1920,
+          height: 1280,
+        },
```
Comment (alannnc) [inline playwright.config.ts:65]: Why this change was needed?

**Review by alannnc — APPROVED** [review]
Code looks good, couldn't tested it properly. Just a note about playwright config being changed.

File: playwright.config.ts (hunk 1 above)
Comment (emrysal) [reply playwright.config.ts]: Suspected that viewport size was the reason the end to end tests were failing which turned out to be wrong, it's back to normal now.
